src/uct/ib/rc/accel/rc_mlx5.h

```diff
@@ -21,6 +21,7 @@
 typedef struct uct_rc_mlx5_iface_config {
     uct_rc_iface_config_t  super;
     unsigned               tx_max_bb;
+    double                 fc_soft_thresh;
```

Comment:
how about creating new struct "uct_rc_fc_config_t { double dc_soft_thresh}" and include it both in rc_verbs and rc_mlx5, and also create config table for this struct and use it - instead of duplicating "FC_SHOFT_THRESH" parameter definition?